--- avrdude-5.3.1/configure.ac.old	2006-12-22 23:55:33.000000000 +0100
+++ avrdude-5.3.1/configure.ac	2007-04-04 20:43:59.000000000 +0200
@@ -157,19 +157,8 @@
 		esac],
 	[enabled_parport=yes])
 
-if test "$enabled_doc" = "yes"; then
-
-SUBDIRS_AC='doc @WINDOWS_DIRS@'
-DIST_SUBDIRS_AC='doc windows'
-else
-
-SUBDIRS_AC='@WINDOWS_DIRS@'
-DIST_SUBDIRS_AC='windows'
-fi
 
 AC_SUBST(DOC_INST_DIR, $DOC_INST_DIR)
-AC_SUBST(SUBDIRS_AC, $SUBDIRS_AC)
-AC_SUBST(DIST_SUBDIRS_AC, $DIST_SUBDIRS_AC)
 
 
 # Find the parallel serial device files based on target system
@@ -236,6 +225,18 @@
 		LDFLAGS="${LDFLAGS} -static"
 		;;
 esac
+
+if test "$enabled_doc" = "yes"; then
+
+SUBDIRS_AC="doc $WINDOWS_DIRS"
+DIST_SUBDIRS_AC='doc windows'
+else
+
+SUBDIRS_AC="$WINDOWS_DIRS"
+DIST_SUBDIRS_AC='windows'
+fi
+AC_SUBST(SUBDIRS_AC, $SUBDIRS_AC)
+AC_SUBST(DIST_SUBDIRS_AC, $DIST_SUBDIRS_AC)
 AC_SUBST(WINDOWS_DIRS,$WINDOWS_DIRS)
 
 # If we are compiling with gcc, enable all warning and make warnings errors.
