diff -up avrdude-8.0/src/doc/CMakeLists.txt._orig avrdude-8.0/src/doc/CMakeLists.txt
--- avrdude-8.0/src/doc/CMakeLists.txt._orig	2024-08-24 22:51:00.000000000 +0200
+++ avrdude-8.0/src/doc/CMakeLists.txt	2024-09-01 11:57:53.844478755 +0200
@@ -130,13 +130,9 @@ add_custom_command(
 # =====================================
 
 add_custom_target(info ALL DEPENDS avrdude.info)
-add_custom_target(pdf ALL DEPENDS avrdude.pdf)
-add_custom_target(html ALL DEPENDS avrdude-html/avrdude.html)
 
 # =====================================
 # Install
 # =====================================
 
 install(FILES "${CMAKE_CURRENT_BINARY_DIR}/avrdude.info" DESTINATION ${CMAKE_INSTALL_INFODIR})
-install(FILES "${CMAKE_CURRENT_BINARY_DIR}/avrdude.pdf" DESTINATION ${CMAKE_INSTALL_DOCDIR})
-install(DIRECTORY "${CMAKE_CURRENT_BINARY_DIR}/avrdude-html" DESTINATION ${CMAKE_INSTALL_DOCDIR})
