--- avrdude-5.5/configure.ac.orig	2007-10-30 00:00:12.000000000 +0100
+++ avrdude-5.5/configure.ac	2008-02-10 00:16:36.000000000 +0100
@@ -240,6 +240,18 @@
 		LDFLAGS="${LDFLAGS} -static"
 		;;
 esac
+
+if test "$enabled_doc" = "yes"; then
+
+SUBDIRS_AC="doc $WINDOWS_DIRS"
+DIST_SUBDIRS_AC='doc windows'
+else
+
+SUBDIRS_AC="$WINDOWS_DIRS"
+DIST_SUBDIRS_AC='windows'
+fi
+AC_SUBST(SUBDIRS_AC, $SUBDIRS_AC)
+AC_SUBST(DIST_SUBDIRS_AC, $DIST_SUBDIRS_AC)
 AC_SUBST(WINDOWS_DIRS,$WINDOWS_DIRS)
 
 # If we are compiling with gcc, enable all warning and make warnings errors.
